docs/pdr_altitudecontrol.md

```diff
@@ -0,0 +1,279 @@
+- Maximum lift gas reserve fraction
+
+## Avionics
+The altitude control flight software application ([Main Flight Computer
+Software Design](../pdr_avsw.md#main-flight-computer-software-design)) issues
+commands to open the ballast or bleed valves over the CAN bus. When a bleed or
+ballast command is received by the altitude control interface board
+([Altitude Control/Dynamics Sensor Card](../pdr_avhw.md#avionics-sensor-card))
+over CAN, it executes the corresponding subroutine to carry out the command.
+The software environment onboard the altitude control interface board
+([Altitude Control/Dynamics Card Software Design](../pdr_avsw.md#avionics-sensor-card-software-design))
+includes drivers for the hardware interfaces, such as a PWM motor controller
+for the valve actuators.
+
+The altitude control interface board is also responsible for polling sensors
+and actuators for telemetry and pre-processing the raw signals (e.g. voltages
+from a pressure sensor) into the appropriate derived values (e.g. values in
+units of pressure) before sending them to the CAN bus for use by other flight
+software and downlink.
+
+![Flight Software Diagram](img/pdr/altitudecontrol/fsw-diagram.png)
+
+### Sensing
+- [Avionics Sensor Card](../pdr_avhw.md#avionics-sensor-card)
+    - GPS altitude
+    - Barometric altitude
+    - Ambient pressure
+- [Balloon Sensor Board](../pdr_avhw.md#balloon-board)
+    - Balloon valve position/state
+    - Balloon internal pressure
+    - Remaining lift gas
+- [Altitude Control/Dynamics Sensor Card](../pdr_avhw.md#altitude-control-dynamics-sensor-card)
+    - Ballast valve position/state
+    - Remaining ballast sensor
+
+### Actuation
+- **Raise altitude:** Decrease total mass by releasing ballast. (See [Ballast Hopper](../pdr_mechanics.md#ballast-hopper))
+- **Lower altitude:** Decrease buoyancy by venting lift gas. (See [Balloon Plug](../pdr_mechanics.md#balloon-plug))
+
+## System Design Parameters
+### Requirements
+
+| Requirement | Description | Target | Allowable Bounds |
+| --- | --- | --- | --- |
+| Settling time | Time for controller to transition from stabilize to cruise mode. | 20 minutes. | 10 minutes - 1 hour |
+| Steady state error (dead zone) | Magnitude of the difference between target altitude and actual altitude during cruise mode | 150 m | 0 m - 500 m |
+| Ballast mass budget | Portion of total system mass budget allocated to ballast material | 1 kg or 25% | 1 kg - 2 kg |
+| Bleed mass budget | Portion of total system lift gas allocated to altitude control. | 0.5 kg or 30% | No more than the mass of lift gas needed for positive lift with empty ballast |
+| Flight duration | Impact on total flight time (launch to recovery) with altitude control enabled compared to an equivalent flight with altitude control disabled | 7 hours | No less than the |modeled flight duration with controller disabled (upper bound constrained by geofencing and weather) |
+
+### System Elements
```

Comment:
These tables don't play to well with the current theme.

Could we transition to a theme that will show these larger tables?